### ISSUE TITLE
[Suggestion 1.13] Make AE and RefinedStorage recipes cheaper

### ISSUE THREAD

**Issue by TheUnderTaker11** [Morpheus1101/Avaritia#130]
They are only able to power the AE or RS storage system respectively, and as far as I am aware there is no other way to pull power out. So besides the fact  you can make the full 128^3 spatial pylon setup easily with creative powercell (Which honestly if you have the resources for an infinity catalyst you can do anyway), these blocks will ONLY power their respective system. You're looking at like, what, around10k rf/t for a huge system? That's basically tossing your infinity ingots in the trash! Depending on what mods are used to get the resources, the amount this would save you in rf/t would be basically negligible.

To sum it up, in my opinion, requiring more than like 4 infinity ingots at most is wayyyy overkill on cost, compared to other methods. Of course, unless I'm missing something and there IS some way to extract power from them. Then obviously just tell me and close this cause that would make my whole point void.

(Also on an unrelated note it's nice seeing my recipes being added into base Avaritia. I have since picked up development and fixed all bugs and such, so will be keeping up with what you add so there isn't duplicates!)

**Comment by Morpheus1101** [comment]
Overlooking the request and seeing as your are literally end game and have got a massive network for either mod and they only power there own respective mods, i have overhauled the recipes(Still have RS to go at time of writing)
